sdk-extensions/resources/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+    `java-library`
+    `maven-publish`
+
+    // TODO(anuraaga): Use reflection for RuntimeMXBean which is not included in Android.
```

Comment:
we need an issue for this, yes?

Reply:
https://github.com/open-telemetry/opentelemetry-java/issues/2815

Reply:
Thanks!